--- ../dwm/config.def.h	2020-11-21 23:37:45.990049172 +0000
+++ config.def.h	2020-12-05 17:05:34.333768406 +0000
@@ -7,15 +7,13 @@
 static const int topbar             = 1;        /* 0 means bottom bar */
 static const char *fonts[]          = { "monospace:size=10" };
 static const char dmenufont[]       = "monospace:size=10";
-static const char col_gray1[]       = "#222222";
-static const char col_gray2[]       = "#444444";
-static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+static const char col_green[]       = "#98971a";
+static const char col_black[]       = "#282828";
+static const char col_grey[]        = "#a89984";
 static const char *colors[][3]      = {
 	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	[SchemeNorm] = { col_grey, col_black, col_grey },
+	[SchemeSel]  = { col_black, col_green,  col_black  },
 };
 
 /* tagging */
@@ -27,8 +25,8 @@
 	 *	WM_NAME(STRING) = title
 	 */
 	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+//	{ "Gimp",     NULL,       NULL,       0,            0,           -1 },
+	{ "Firefox",  NULL,       NULL,       0,            0,           -1 },
 };
 
 /* layout(s) */
@@ -56,13 +54,48 @@
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
-static const char *termcmd[]  = { "st", NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_black, "-nf", col_grey, "-sb", col_green, "-sf", col_black, NULL };
+static const char *terminal[]  = { "st", NULL };
+static const char *htop[] = { "st", "htop", NULL };
+static const char *filemanager[] = { "st", "lf", NULL };
+static const char *vim[] = { "st", "vim", NULL };
+static const char *newsboat[] = { "st", "newsboat", NULL };
+static const char *musicclient[] = { "st", "ncmpcpp", NULL };
+static const char *torrentclient[] = { "st", "transmission-remote-cli", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
+	/* Spawn Controls                                                   */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
-	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
+	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = terminal } },
+	{ MODKEY,                       XK_e,      spawn,          {.v = filemanager } },
+	{ MODKEY,                       XK_z,      spawn,          {.v = htop } },
+	{ MODKEY,                       XK_n,      spawn,          {.v = newsboat } },
+	{ MODKEY,                       XK_v,      spawn,          {.v = vim } },
+	{ MODKEY,                       XK_w,      spawn,          SHCMD("$BROWSER") },
+	{ MODKEY|ControlMask,           XK_w,      spawn,          SHCMD("firefox") },
+	{ MODKEY,                       XK_g,      spawn,          SHCMD("gimp") },
+	{ MODKEY|ControlMask,           XK_t,      spawn,          {.v = torrentclient } },
+	{ MODKEY|ControlMask,           XK_v,      spawn,          SHCMD("virt-manager") },
+	{ MODKEY|ControlMask,           XK_m,      spawn,          {.v = musicclient } },
+	/* Audio Controls */ 
+	{ MODKEY|ShiftMask,             XK_i,      spawn,          SHCMD("amixer set Master 5%+") },
+	{ MODKEY|ShiftMask,             XK_d,      spawn,          SHCMD("amixer set Master 5%-") },
+	{ MODKEY|ShiftMask,             XK_m,      spawn,          SHCMD("amixer -D pulse set Master toggle") },
+	/* Music Controls (mpd) */
+	/* Screencap Controls */
+	{ MODKEY,                       XK_c,      spawn,          SHCMD("dmenu_record")  },
+	{ MODKEY|ShiftMask,             XK_c,      spawn,          SHCMD("dmenu_record end") },
+	{ MODKEY,                       XK_s,      spawn,          SHCMD("maim ~/pictures/screenshots/$(date +%d%m%Y-%H%M%S).jpg") },
+	{ MODKEY|ShiftMask,             XK_s,      spawn,          SHCMD("maim -s ~/pictures/screenshots/$(date +%d%m%Y-%H%M%S).jpg") },
+	{ MODKEY|ControlMask,           XK_k,      spawn,          SHCMD("screenkey") },
+	/* Other Controls */
+	{ MODKEY|ControlMask,           XK_l,      spawn,          SHCMD("slock") },
+	{ MODKEY,                       XK_apostrophe, spawn,      SHCMD("dmenu_VPN") },
+	{ MODKEY,                       XK_semicolon,  spawn,      SHCMD("dmenu_SSH") },
+	{ MODKEY,                       XK_slash,  spawn,          SHCMD("dmenu_scripts") },
+	{ MODKEY,                       XK_y,      spawn,          SHCMD("dmenu_ytd") },
+	{ MODKEY,                       XK_r,      spawn,          SHCMD("dmenu_display") },
 	{ MODKEY,                       XK_b,      togglebar,      {0} },
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
 	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } },
@@ -72,7 +105,7 @@
 	{ MODKEY,                       XK_l,      setmfact,       {.f = +0.05} },
 	{ MODKEY,                       XK_Return, zoom,           {0} },
 	{ MODKEY,                       XK_Tab,    view,           {0} },
-	{ MODKEY|ShiftMask,             XK_c,      killclient,     {0} },
+	{ MODKEY,                       XK_q,      killclient,     {0} },
 	{ MODKEY,                       XK_t,      setlayout,      {.v = &layouts[0]} },
 	{ MODKEY,                       XK_f,      setlayout,      {.v = &layouts[1]} },
 	{ MODKEY,                       XK_m,      setlayout,      {.v = &layouts[2]} },
@@ -103,7 +136,6 @@
 	{ ClkLtSymbol,          0,              Button1,        setlayout,      {0} },
 	{ ClkLtSymbol,          0,              Button3,        setlayout,      {.v = &layouts[2]} },
 	{ ClkWinTitle,          0,              Button2,        zoom,           {0} },
-	{ ClkStatusText,        0,              Button2,        spawn,          {.v = termcmd } },
 	{ ClkClientWin,         MODKEY,         Button1,        movemouse,      {0} },
 	{ ClkClientWin,         MODKEY,         Button2,        togglefloating, {0} },
 	{ ClkClientWin,         MODKEY,         Button3,        resizemouse,    {0} },
